app/presenters/content_row_presenter.rb

```diff
@@ -3,26 +3,34 @@ class ContentRowPresenter
   include ActiveSupport::Inflector
 
   attr_reader :title, :base_path, :document_type, :upviews,
-              :user_satisfaction, :searches, :raw_document_type
+              :user_satisfaction_responses, :user_satisfaction_percentage, :searches, :raw_document_type
   def initialize(data)
     @title = data[:title]
     @base_path = format_base_path(data[:base_path])
     @raw_document_type = data[:document_type]
     @document_type = humanize(data[:document_type])
     @upviews = number_with_delimiter(data[:upviews], delimiter: ',')
-    @user_satisfaction = format_satisfaction(
+    @user_satisfaction_percentage = format_satisfaction_percentage(data[:satisfaction])
+    @user_satisfaction_responses = format_satisfaction_responses(
       data[:satisfaction], data[:useful_yes], data[:useful_no]
     )
+
     @searches = number_with_delimiter(data[:searches], delimiter: ',')
   end
 
 private
 
-  def format_satisfaction(score, yes_responses, no_responses)
+  def format_satisfaction_percentage(score)
     return 'No responses' unless score
```

Comment:
Could we have `format_satisfaction_responses` method return this? Currently its confusing what each method should return. Instead this method could be like 
```suggestion
    return "#{(score * 100).round(0)}%" if score
```
and it would work with the suggested changes to the view below.